lms/static/js/learner_dashboard/views/course_card_view.js

```diff
@@ -28,6 +30,18 @@
                 render: function() {
                     var filledTemplate = this.tpl(this.model.toJSON());
                     HtmlUtils.setHtml(this.$el, filledTemplate);
+                    this.postRender();
+                },
+
+                postRender: function(){
+                    HtmlUtils.setHtml(this.$el.find('.course-actions'),
+                        HtmlUtils.HTML(
+                            new CourseCardActions({
+                                model: this.model,
+                                context: this.context
+                            }).$el.html()
+                        )
+                    );
```

Comment:
I tried to pass in the class name reference for the CourseCardAction view to render, but the view just won't render. I found that it was because at this point, the course card view is not yet rendered unto the DOM because the collection list view would only render to the DOM after all the item views are instantiated.